modules/dex/examples/claim.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cfg.example.org/v1alpha1
```

Comment:
Clean up the existing examples and add your own?

Reply:
Yes, this was the unasked question, what's next? 😄 But as noted, this was mostly about me learning the basics and getting to the point where actual work can start, so it doesn't look like much.